tests/unit/src/main/scala/tests/codeactions/BaseCodeActionLspSuite.scala

```diff
@@ -54,19 +56,31 @@ abstract class BaseCodeActionLspSuite(
       changeFile: String => String = identity,
       expectError: Boolean = false,
       filterAction: CodeAction => Boolean = _ => true,
+      scalaCli: Boolean = false,
+      skipDiff: Boolean = false,
```

Comment:
```suggestion
      assertion:  Option[(Boolean, String)],
```
Maybe something like this? Have separate assertion to make sure `skipDiff` is not used by mistake?

Reply:
I don't understand why you used Boolean there, so I changed the assertion type a bit to:
```
assertion:  Option[Unit => Unit]
```
which allows to override default assert in test

Reply:
I was thinking:
```
assertion.map{ case (condition, msg) => assert(condition, msg)}
```
This way we would steer with the types what needs to be done.